src/components/footer.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { useHistory } from 'react-router-dom';
+
+const StyledFooter = styled.footer`
+  background-color: var(--black);
+  padding: 1.5rem 1.25rem;
+
+  ul {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+
+    li:first-of-type {
+      margin-right: 1rem;
+    }
+  }
+
+  a {
+    color: var(--white);
+  }
+`;
+
+const Navigation: React.FC = () => {
+  const [isHome, setIsHome] = useState(true);
+  const {
+    location: { pathname },
+  } = useHistory();
+
+  useEffect(() => {
+    pathname === '/about' ? setIsHome(false) : setIsHome(true);
```

Comment:
```suggestion
    setIsHome(pathname !== '/about');
```

or maybe `pathname === '/'`

Reply:
Nice one!